src/target/llvm/llvm_module.cc

```diff
@@ -356,6 +356,26 @@ TVM_REGISTER_GLOBAL("codegen.build_llvm")
     *rv = runtime::Module(n);
   });
 
+TVM_REGISTER_GLOBAL("codegen.LLVMModuleCreate")
+.set_body([](TVMArgs args, TVMRetValue *rv) {
+  auto n = make_object<LLVMModuleNode>();
+  auto target = args[0].operator std::string();
+  auto module_name = args[1].operator std::string();
+
+  // create a default data layout
```

Comment:
Sorry for later response. Minor comment: The logic here doesn't only create default data layout but also create default target triple. We should update the comment.

Reply:
Thanks @FrozenGene I've updated the comments based on your comments :)